tensorflow/python/pywrap_mlir.py

```diff
@@ -106,24 +106,4 @@ def experimental_run_pass_pipeline(mlir_txt, pass_pipeline, show_debug_info):
 
 
 def experimental_write_bytecode(filename, mlir_txt):
-  return ExperimentalWriteBytecode(filename.encode('utf-8'), mlir_txt.encode())
-
-
-def experimental_tflite_to_tosa_bytecode(
-    flatbuffer,
-    bytecode,
-    use_external_constant=False,
-    ordered_input_arrays=None,
-    ordered_output_arrays=None,
-):
-  if ordered_input_arrays is None:
-    ordered_input_arrays = []
-  if ordered_output_arrays is None:
-    ordered_output_arrays = []
-  return ExperimentalTFLiteToTosaBytecode(
-      flatbuffer.encode('utf-8'),
-      bytecode.encode('utf-8'),
-      use_external_constant,
-      ordered_input_arrays,
-      ordered_output_arrays,
-  )
+  return ExperimentalWriteBytecode(filename.encode('utf-8'), mlir_txt.encode())
```

Comment:
This line should not change. Please configure your editor to insert a newline at the end.
```suggestion
  return ExperimentalWriteBytecode(filename.encode('utf-8'), mlir_txt.encode())

```